tensorboard/webapp/plugins/npmi/views/annotations_list/annotation/annotation_component.ts

```diff
@@ -45,11 +46,16 @@ export class AnnotationComponent implements AfterViewInit, OnChanges {
   @Input() showCounts!: boolean;
   @Input() annotation!: string;
   @Input() runHeight!: number;
+  @Input() sidebarWidth!: number;
```

Comment:
I don't see sidebarWidth used anywhere. Is its sole purpose to trigger ChangeDetection on the component? If so, consider adding a comment that says so (and in the other component, too).